packages/framer-motion/package.json

```diff
@@ -73,31 +73,31 @@
     "bundlesize": [
         {
             "path": "./dist/size-rollup-motion.js",
-            "maxSize": "29.40 kB"
+            "maxSize": "29.12 kB"
```

Comment:
This one is the "real" saving - the others saw a similar decrease with the `class` PR and are likely from a previous PR.